test/test_pattern_matcher.py

```diff
@@ -17,6 +17,56 @@ def test_simple_match(self):
     self.assertEqual(matcher.rewrite(c1), c1)
     self.assertEqual(matcher.rewrite(c2), None)
 
+  def test_uop(self):
+    matcher = PatternMatcher([({"__name__": "x", "uop": UOps.CONST}, lambda x: x)])
+    c1 = UOp(UOps.CONST)
```

Comment:
please check the UOp spec https://github.com/tinygrad/tinygrad/blob/master/tinygrad/codegen/uops.py#L398 to create a correct UOp.

Reply:
thanks, will fix this later today

Reply:
done!